crates/compiler/builtins/roc/Dict.roc

```diff
@@ -34,7 +34,7 @@ interface Dict
 ## A [dictionary](https://en.wikipedia.org/wiki/Associative_array) that lets you
 ## associate keys with values.
 ##
-## ### Inserting
+## **Inserting**
```

Comment:
Can you explain this change?

Reply:
I see, I do recommend we change it back to `###` in the future. In general, style should not be modified by changing  structural elements. It's a bit like changing an <h2></h2> to a <b></b> in HTML, it may look good but now we are no longer properly representing the structure of your page.

Reply:
I have updated the css to correctly handle headings. I've updated the "Performance Details" headings. I haven't reverted all of them back as they need a general pass through to clean up comments and examples, and I would like to do that in another PR.